app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -91,7 +91,7 @@ public function store(Request $request, $seatId = null, $backTo = null)
         // Was the asset updated?
         if ($licenseSeat->save()) {
 
-            event(new CheckoutableCheckedIn($license, $return_to, Auth::user(), $request->input('note')));
```

Comment:
I think this is from a previous PR, right? Not sure why it's here.

Reply:
It is... from before this commit: https://github.com/snipe/snipe-it/pull/6619/commits/866ec0f5703775180e8106768da64ceeec279c7d, I think.

Reply:
The checkin.blade.php change also shows up, and its part of the previous PR. I think those are already merged?

Reply:
They are, so I'm not sure why they're showing up here...